README.md

```diff
@@ -0,0 +1,85 @@
+# Kubernetes Monitor Message Contracts
+This repository contains the Protobuf contracts and the generated gRPC SDKs that are used for communication between Octopus Server and the Octopus Kubernetes Monitor.
+
+# Guidelines
+
+## General
+- This repository is set up with [release please](https://github.com/googleapis/release-please) to handle versioning, changelogs and releasing. To ensure that the changelogs are generated correctly, please use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) when merging a PR into `main`.
```

Comment:
Might be worthwhile setting up the action to enforce the conventional commit syntax for PR names like we have on the monitor?

Reply:
Good idea, ill add that in a separate PR